keiko/keiko/keiko.py

```diff
@@ -121,24 +121,16 @@ def generate_report(
             tex_output_file_name,
         ]
         env = {**os.environ, "TEXMFVAR": tmp_dirname}
-        output = subprocess.run(cmd, cwd=tmp_dirname, env=env, capture_output=True, check=False)
-        logger.info("pdflatex run. [report_id=%s] [template=%s] [command=%s]", report_id, template_name, " ".join(cmd))
-        logger.debug(output.stdout.decode("utf-8"))
-        logger.debug(output.stderr.decode("utf-8"))
-        if output.returncode:
-            raise Exception("Error running pdflatex")
-
-        output = subprocess.run(cmd, cwd=tmp_dirname, env=env, capture_output=True, check=False)
-        logger.info(
-            "pdflatex run. [report_id=%s] [template=%s] [command=%s]",
-            report_id,
-            template_name,
-            " ".join(cmd),
-        )
-        logger.debug(output.stdout.decode("utf-8"))
-        logger.debug(output.stderr.decode("utf-8"))
-        if output.returncode:
-            raise Exception("Error running pdflatex")
+        for i in range(2):
```

Comment:
May be nicer to start with 1
```suggestion
        for i in enumerate(range(2), 1):
```
I'll leave it up to you!